lib/request-handler.spec.js

```diff
@@ -20,10 +20,10 @@ const baseUri = `http://127.0.0.1:${config.port}`;
 function performTwoRequests (first, second) {
   return fetch(`${baseUri}${first}`)
     .then(res => {
-      assert.ok(res.ok);
+      expect(res.ok).to.equal(true);
```

Comment:
Can we have `expect(res.ok).to.be.true;` here? Is is better than `expect(res.ok).to.equal(true);` for you?

Reply:
Yea, great! I didn't realize that was a thing. There are a few of these to update.